app/views/coronavirus_pages/_sub_sections_summary_list.html.erb

```diff
@@ -17,6 +17,10 @@
 <div class="covid-manage-page__accordion-summary-list">
   <%= render "govuk_publishing_components/components/summary_list", {
     title: @coronavirus_page.sections_title,
-    items: sub_sections
+    items: sub_sections,
+    edit: {
+      link_text: "Reorder",
+      href: reorder_coronavirus_page_sub_sections_path(@coronavirus_page.slug,)
```

Comment:
That's last comma looks a bit odd. If it's for the "every element in a hash ends with a comma - even the last one" syntax, the comma should be outside the bracket. I suggest:
```ruby
href: reorder_coronavirus_page_sub_sections_path(@coronavirus_page.slug),
```
Or leave it off completely